src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -2231,8 +2230,13 @@ static int xgq_collect_sensors(struct platform_device *pdev, int aid, int sid,
 	u32 length = 0;
 	int ret = 0;
 	int id = 0;
+	struct xgq_cmd_cq_vmr_payload *vmr_status = (struct xgq_cmd_cq_vmr_payload *)&xgq->xgq_cq_payload;
```

Comment:
why remove a function and add dup code here?

Reply:
vmr_status_query payload gets information of both has_ext_scfw and sc_is_ready in a single call, so changed code to optimize functionality

Reply:
I agree @karthdmg-xilinx , We shouldnt be removing the function. Better to rename to the function to get the sc status, <not available, ready, not ready> enums.

Reply:
@chvamshi-xilinx @xdavidz uploaded changes as discussed, kindly review